event_listener/dfusion_db/event_pusher.py

```diff
@@ -0,0 +1,43 @@
+from pymongo import MongoClient
+from django.conf import settings
+
+client = MongoClient(
+    host=settings.MONGO_HOST,
+    port=settings.MONGO_PORT
+)
+db = client.get_database(settings.DB_NAME)
+
+
+def post_deposit(event: dict):
+    """
+    :param event: dict
+    :return: bson.objectid.ObjectId
+    """
+    # Verify integrity of post data
+    assert event.keys() == {'accountId', 'tokenId', 'amount', 'slot'}, "Unexpected Event Keys"
+    assert all(isinstance(val, int) for val in event.values()), "One or more of deposit values not integer type"
+
+    deposits = db.deposits
+    deposit_id = deposits.insert_one(event).inserted_id
+    return deposit_id
```

Comment:
Maybe it's not supposed to be this layer, but I'd hope we could have one component that abstracts all the db specific code (in this case the fact that we are using mongoDb) away. I'm not sure if that layer should do any validation (e.g. the asserts above).

It would also require to return database agnostic return types (in this example a success boolean vs. deposit_id), in the case where we fetch data the raw json or a data model object.

Reply:
Although, I would prefer if we make this part of the database infrastructure a proper issue and handle it in a separate branch.